src/buildblock/ProjData.cxx

```diff
@@ -289,6 +289,25 @@ ProjData::get_related_viewgrams(const ViewSegmentNumbers& view_segmnet_num,
   return RelatedViewgrams<float>(viewgrams, symmetries_used);
 }
 
+std::vector<float>
+ProjData::get_related_bin_values(const std::vector<Bin>& r_bins) const
```

Comment:
do we need it? If not, comment out. I'd rather have a RelatedBins interface later.